src/db/webhooks/getAllWebhooks.ts

```diff
@@ -3,6 +3,9 @@ import { prisma } from "../client";
 
 export const getAllWebhooks = async (): Promise<Webhooks[]> => {
   return await prisma.webhooks.findMany({
+    where: {
```

Comment:
We never care about deleted webhooks so omit it in the query to simplify business logic.